controllers/util/secrets_util.go

```diff
@@ -534,18 +640,26 @@ func (sutil *SecretsUtil) ensureS3SecretResources(secretName, namespace string)
 	// Cleanup policy if secret is deleted
 	sutil.Log.Info("Cleaning up secret policy", "secret", secretName)
 
-	return nil, sutil.deletePolicyResources(&secret, namespace)
+	return nil, sutil.deletePolicyResources(&secret, namespace, format)
 }
 
-func (sutil *SecretsUtil) AddSecretToCluster(secretName, clusterName, namespace, targetns string) error {
-	sutil.Log.Info("Add Secret", "cluster", clusterName, "secret", secretName)
+func (sutil *SecretsUtil) AddSecretToCluster(
+	secretName, clusterName, namespace, targetNS string,
+	format TargetSecretFormat,
+	veleroNS string,
```

Comment:
accepting targetNS and velroNS is not consistent.

Is verlroNS another variant of targetNS?